ramalama/cli.py

```diff
@@ -721,6 +721,15 @@ def run_parser(subparsers):
     parser.add_argument(
         "ARGS", nargs="*", help="Overrides the default prompt, and the output is returned without entering the chatbot"
     )
+    # Disable network access by default, and give the option to pass any supported network mode into
+    # podman if needed:
+    # https://docs.podman.io/en/latest/markdown/podman-run.1.html#network-mode-net
+    parser.add_argument(
+        "--network-mode",
+        type=str,
+        default="none",
+        help="Set the network mode for the container.",
```

Comment:
This needs to be documented in man pages.  I do not know why the man page checker allowed this through.

Reply:
I believe the new options should be documented now.